### PR TITLE
Release v6.0.0

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

https://folio-org.atlassian.net/browse/UIF-515
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by BogdanDenis — APPROVED** [review]

**Review by zburke — CHANGES_REQUESTED** [review]
The PR title here is "Release v6.0.0" but the content of this PR bumps the version to `6.1.0`; which is correct, the code or the documentation?

[`stripes-acq-components` `5.1.0` was just published for Q](https://github.com/folio-org/stripes-acq-components/releases/tag/v5.1.0). The `6.0.0` release of ui-finance should depend on that version, but you're not bumping to `5.1.0` until this PR, where the version is `6.1.0`. 

I'm not really sure how to move forward from here. Since this PR is trying to do so many incompatible things at once, the easiest path is probably just to close this and then open separate PRs for each of the separate concerns:
1. correctly set the stripes-acq-components version for the 6.0.0 release
2. [edit] correctly set the stripes-data-transfer-components version for the 6.0.0 release, as noted by @UladzislauKutarkin below
3. release v6.0.0
4. bump the version to v6.1.0

**Comment by zburke** [comment]
PS: If you're not going to use the PR template, please remove it. An empty to-do list is just noise.

File: package.json [hunk 1]
```diff
@@ -466,7 +466,7 @@
     "regenerator-runtime": "^0.13.3"
   },
   "dependencies": {
-    "@folio/stripes-acq-components": "~5.0.0",
+    "@folio/stripes-acq-components": "~5.1.0",
     "@folio/stripes-data-transfer-components": "~6.0.0",
```
Comment (UladzislauKutarkin) [inline package.json:470]: Why you haven't bumped `stripes-data-transfer-components`? It's already released

Comment (alisher-epam) [reply package.json:470]: @UladzislauKutarkin Because The module was not released when I created the PR.

**Comment by alisher-epam** [comment]
Will be creating a separate PR

**Pull request closed without merging**